src/PKSim.Presentation/Presenters/Individuals/ExpressionParametersPresenter.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using OSPSuite.Presentation.DTO;
+using OSPSuite.Presentation.Presenters;
+using OSPSuite.Utility.Extensions;
+using PKSim.Core.Model;
+using PKSim.Presentation.DTO.Individuals;
+using PKSim.Presentation.Presenters.Parameters;
+using PKSim.Presentation.Services;
+using PKSim.Presentation.Views.Individuals;
+
+namespace PKSim.Presentation.Presenters.Individuals
+{
+   public interface IExpressionParametersPresenter:IPresenter<IExpressionParametersView>, IEditParameterPresenter
+   {
+      void Edit(IReadOnlyList<ExpressionParameterDTO> expressionParameters);
+      void SetExpressionParameterValue(IParameterDTO expressionParameterDTO, double value);
+
+   }
+
+   public class ExpressionParametersPresenter : EditParameterPresenter<IExpressionParametersView, IExpressionParametersPresenter>, IExpressionParametersPresenter
```

Comment:
I Am reusing this presenter in the simulation. IT was not possible before because we had a different structure. But it's now the same :)